fdbclient/DatabaseContext.h

```diff
@@ -140,6 +140,20 @@ class ClientTagThrottleData : NonCopyable {
 	}
 };
 
+class WatchMetadata: public ReferenceCounted<WatchMetadata> {
+	public:
```

Comment:
I'm going to be a bit picky here style-wise -- to match the style of the rest of the codebase, `public:` should not be indented and the fields in `WatchMetadata` should be an indentation level 1.